src/nlsolve_ac_powerflow.jl

```diff
@@ -153,5 +162,100 @@ function _nlsolve_powerflow(data::ACPowerFlowData; nlsolve_kwargs...)
     if !res.f_converged
         @error("The powerflow solver returned convergence = $(res.f_converged)")
     end
-    return res
+    return res.f_converged, res.zero
+end
+
+function _nlsolve_powerflow(pf::KLUACPowerFlow, data::ACPowerFlowData; nlsolve_kwargs...)
+    pf = PolarPowerFlow(data)
+    #J_function = PowerFlows.PolarPowerFlowJacobian(data, pf.x0)
+
+    maxIter = 30  # TODO
+    tol = 1e-6    # TODO
+    i = 0
+
+    Vm = data.bus_magnitude[:]
+    Va = data.bus_angles[:]
+    V = Vm .* exp.(1im * Va)
+
+    # Find indices for each bus type
+    ref = findall(x -> x == PowerSystems.ACBusTypesModule.ACBusTypes.REF, data.bus_type)
+    pv = findall(x -> x == PowerSystems.ACBusTypesModule.ACBusTypes.PV, data.bus_type)
+    pq = findall(x -> x == PowerSystems.ACBusTypesModule.ACBusTypes.PQ, data.bus_type)
```

Comment:
Probably no significant difference, but note that these three lines could be combined to only iterate over the bus types once